### PR TITLE
chore: update README.md

### PR DESCRIPTION
**Pull request by DieHard073055**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,40 @@
 # solid-carnival
```
Comment (pamdayne) [inline README.md]: Is this line needed?

File: README.md
Comment (pamdayne) [inline README.md:1]: If L1 is not needed, we can remove that.
Then suggest to replace all headings to use Heading1 (or single #) for easy readability for sectioning.

Comment (DieHard073055) [reply README.md:1]: done

File: README.md (hunk 1 above)
Comment (DieHard073055) [reply README.md]: no

File: README.md
Comment (pamdayne) [reply README.md:1]: Thank you!

**Review by pamdayne — APPROVED** [review]

**Pull request merged**